### PR TITLE
Add running Status support

### PR DESCRIPTION
**Pull request by hchoura**

### CODE REVIEW

File: src/foam/nanos/script/Script.js [hunk 1]
```diff
@@ -324,6 +324,12 @@ foam.CLASS({
         return getId();
       `
     },
+    function clone() {
```
Comment (ghost) [inline src/foam/nanos/script/Script.js]: these functions don't make sense - if we're cloning, then why are we just returning `this` immediately? they also don't seem to be used by this PR...

Comment (jlhughes) [reply src/foam/nanos/script/Script.js]: @petenanopay these override the defaults in FObject.java which do copy.  We are explicitly not copying in this case.

Comment (kgrgreer) [reply src/foam/nanos/script/Script.js]: @hchoura Are these two clone() functions required to fix the problem? If so, then the issue is that the Controller is now clone()-ing, which is breaking this feature. We should fix it there and leave out these methods. These were just to test if the problem was clone related.

Comment (hchoura) [reply src/foam/nanos/script/Script.js]: I retest it. We can delete them.

**Pull request merged**